admin/howto/create-billing-account.md

```diff
@@ -0,0 +1,73 @@
+# Provision a paid cloud provider account for 2i2c
+
+For 2i2c to build & manage your JupyterHub, it needs *full* access to a
+cloud provider account on [Google Cloud Platform](https://cloudn.google.com),
+[Amazon Web Services][https://aws.amazon.com], or [Azure](https://azure.microsoft.com).
+This page will provide guidance on how to do that.
+
+## Google Cloud Platform
+
+Currently, Google Cloud Platform (GCP) is 2i2c's preferred cloud provider. 2i2c
+access can be set up in a couple different ways.
+
+### Full billing account access
+
+A GCP [billing account](https://cloud.google.com/billing/docs/how-to/manage-billing-account)
+is attached to a source of funds (Credit Card, Institutional account with invoicing,
+credits provided by Google, etc). Giving 2i2c full access to a billing account
+lets us manage everything except the funding mechanism. You can track costs,
+get invoices, and grant overall access by controlling the billing account.
+
+1. [Create a billing account](https://cloud.google.com/billing/docs/how-to/manage-billing-account#create_a_new_billing_account)
+   This should be fairly straightforward with a credit card.
+   
+2. Give 2i2c [billing administrator](https://cloud.google.com/billing/docs/how-to/billing-access)
+   permissions. You can do this by:
+   
+   1. Going to 'Account Management' on the left menu inside your billing account
+      page.
+   2. Click 'Show Info Panel' on the right. This should show a right sidebar that
+      has list of current people with access to this billing account.
+   3. Click 'Add Member'. This opens another right sidebar that lets you add /
+      remove billing administrators.
+   4. Select 'Billing -> Billing account administrator' as the role.
+      
+      ```{note}
+      If you want 2i2c to not see anything about your payment method, select
+      the following roles instead: 'Billing -> Billing Accounts Costs Manager',
+      'Billing -> Billing Accounts User'. The full administrator role is 
+      preferred.
+      ```
+   5. Enter the following user emails under 'New members': `yuvipanda@2i2c.org`,
+      `choldgraf@2i2c.org`. These are two of the folks at 2i2c who will
+      help manage your project.
+   6. Click 'Save'. This sends 2i2c staff an invitation to your billing account.
+   
+   Let 2i2c staff know once you've completed this process, and they should be
```

Comment:
Perhaps we should give them a more specific instruction. Assuming that we have got an issue open per hub, maybe we should tell them to ping us on that issue? (and we should build into the template for hub issues whether we need them to provision an account)

Reply:
Yeah, that makes sense. I'm not sure of the exact mechanism here though. Shall we open an issue for that?

Reply:
Yeah let’s add this to a goal, don’t we have a goal for something like “define communication pathways for champions”?

Reply:
I opened https://github.com/2i2c-org/pilot-hubs/issues/306#issuecomment-808767917 for it.